Jasmine/Common/Models/Coordinate.swift

```diff
@@ -34,3 +34,17 @@ extension Coordinate: Equatable {
             && lhs.col == rhs.col
     }
 }
+
+extension Coordinate: Comparable {
+    /// Compares two Coordinates. Comparison is done by looking at the row, then by column.
+    ///
+    /// - Parameter other: the other coordinate to be compared
+    /// - Returns: true if and only if lhs < rhs
+    static func < (_ lhs: Coordinate, _ rhs: Coordinate) -> Bool {
+        if lhs.row == rhs.row {
+            return lhs.col < rhs.col
+        } else {
+            return lhs.row < rhs.row
```

Comment:
Anything goes for me. The ternary operator will make it more concise though so I shall just go with it